alibi/explainers/integrated_gradients.py

```diff
@@ -481,3 +483,42 @@ def build_explanation(self,
         data.update(deltas=deltas)
 
         return Explanation(meta=copy.deepcopy(self.meta), data=data)
+
+
+class IntegratedGradientsText(Explainer):
+
+    def __init__(self,
+                 model: Union[tf.keras.Model, 'keras.Model'],
+                 layer: Union[None, tf.keras.layers.Layer, 'keras.layers.Layer'] = None,
+                 method: str = "gausslegendre",
+                 n_steps: int = 50,
+                 internal_batch_size: Union[None, int] = 100,
+                 text_transformer: TextTransformer = None,
+                 ) -> None:
+        self._explainer = IntegratedGradients(model=model,
+                                              layer=layer,
+                                              method=method,
+                                              n_steps=n_steps,
+                                              internal_batch_size=internal_batch_size)
+        self.text_transformer = text_transformer
+
+    def explain(self,
+                X: List[str],
+                baselines: Union[None, int, float, np.ndarray] = None,
+                target: Union[None, int, list, np.ndarray] = None) -> Explanation:
+        tokens = self.text_transformer.texts_to_tokens(X)
+        X = self.text_transformer.texts_to_array(X)
+
+        # TODO: baselines assumed to be wrt to the pre-processed X
+        explanation = self._explainer.explain(X=X, baselines=baselines, target=target)
+
+        # exctract data, augment, and create a new explanation object
+        # NB: we have to create a new object for attribute access to work for the new attributes
+        meta, data = explanation.meta, explanation.data
+        attrs = data['attributions'].sum(axis=-1)  # TODO: dangerous to assume these are for embeddings!?
```

Comment:
Seems like a good default for the vast majority of the cases. Maybe add an optional kwarg if the layer is not an embedding layer, then can pass a Callable which applies the correct operation to get your attributions?